test/skate_web/controllers/detours_controller_test.exs

```diff
@@ -134,7 +134,7 @@ defmodule SkateWeb.DetoursControllerTest do
     test "fetches detours from database and groups by active, past, draft", %{conn: conn} do
       author_id = populate_db_and_get_user(conn)
 
-      conn = get(conn, "/api/detours/get_detours")
+      conn = get(conn, "/api/detours")
```

Comment:
Maybe not for this PR, but I think it's considered best practice in Phoenix to refer to URL's with `~p`, so like `conn = get(conn, ~p"/api/detours")`

Non-blocking

Reply:
Not for this PR because it looks like it was already this way.